test/transform/split-declaration.js

```diff
@@ -0,0 +1,70 @@
+import createTestHelpers from '../createTestHelpers';
+const {expectTransform} = createTestHelpers(['split-declaration']);
+
+describe('Split Declaration', () => {
+  describe('with only uninitialized variables', () => {
+    it('should split into separate declarations', () => {
+      expectTransform(
+        'var x,y;'
+      ).toReturn(
+        'var x;\nvar y;'
+      );
+    });
+  });
```

Comment:
I think we should also add a sanity-test for the single-variable case that we don't expect to be transformed.